chef/data_bags/crowbar/migrate/network/011_add_ovs_attributes.rb

```diff
@@ -0,0 +1,44 @@
+def upgrade(ta, td, a, d)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for upgrade is too high. [55.23/50]